packages/platform-android/src/commands/runAndroid/index.js

```diff
@@ -296,6 +314,11 @@ function startServerInNewWindow(port, terminal, reactNativePath) {
   if (/^win/.test(process.platform)) {
     procConfig.detached = true;
     procConfig.stdio = 'ignore';
+    //Temporary fix for #484. See comment on line 254
+    fs.writeFileSync(launchPackagerScript, launchPackagerScriptContent, {
```

Comment:
fs is not defined. Run CLI with --verbose flag for more details.
ReferenceError: fs is not defined

Reply:
Are you editing the node_modules files directly?

Reply:
ok, noted
Thank you for the comment